setup.cfg

```
@@ -43,6 +43,8 @@ zip_safe = False
 [options.extras_require]
 brotli =
     Brotli
+asgi =
+    aiofile >=3.0, <4.0
```

no upper bound - https://iscinumpy.dev/post/bound-version-constraints/

```suggestion
    aiofile >=3.0
```